docs/source/release_notes/release-notes.rst

```diff
@@ -80,7 +82,9 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1727>`__)
 
     *    - |fixed| |improved|
-         - Cassandra depedencies have been bumped to newer versions; should fix a bug (#1654) that caused Atlas probing downed Cassandra nodes every few minutes to see if they were up and working yet to eventually take out the entire cluster by steadily building up leaked connections, due to a bug in the underlying driver.
+         - Cassandra depedencies have been bumped to newer versions; should fix a bug (`#1654 <https://github.com/palantir/atlasdb/issues/1654>`__) that caused
+           Atlas probing downed Cassandra nodes every few minutes to see if they were up and working yet to eventually take out the entire cluster by steadily
```

Comment:
The description of the bug isn't the cleanest, but I don't think it's that big of a deal. We should highlight what the  version is that we actually bumped to, and we should just pick one of "fixed" or "improved" (I vote for "fixed").